pcsx2/SPU2/Mixer.cpp

```diff
@@ -679,19 +657,13 @@ __forceinline
 	}
 	else
 	{
-		Out.Left = MulShr32(Out.Left, Cores[1].MasterVol.Left.Value);
-		Out.Right = MulShr32(Out.Right, Cores[1].MasterVol.Right.Value);
+		Out.Left = ApplyVolume(Out.Left, Cores[1].MasterVol.Left.Value);
+		Out.Right = ApplyVolume(Out.Right, Cores[1].MasterVol.Right.Value);
```

Comment:
The previous use of MulShr32 here seems like a bug, properly applying volume here is what causes the big difference in final output volume between this PR and master.